src/eureka/S5_lightcurve_fitting/models/FleckModel.py

```diff
@@ -0,0 +1,328 @@
+import numpy as np
+import astropy.units as unit
+import inspect
+
+import fleck
+
+from .Model import Model
+from ..limb_darkening_fit import ld_profile
+from ...lib.split_channels import split
+
+
+class PlanetParams():
```

Comment:
Rather than copying this class, just import it from `BatmanModels` as was done in `PoetModels.py`

Reply:
Changing to BatmanModels for now so I can run local tests, but leaving a note to myself to check when resolving conflicts that I change it to import from AstroModels instead.